perf/terraform/remote/terraform_override.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+  backend "s3" {
+    bucket         = "terraform-tfstate"
+    key            = "github.com/libp2p/test-plans/perf/terraform/remote/terraform.tfstate"
+    region         = "us-west-2"
+  }
+}
```

Comment:
> Where would the .tfstate file of the long-lived terraform-provisioned infrastructure live?

For IPDX terraforms, we usually store them on S3. We can create an S3 bucket in libp2p account and use it for tfstate storage. I added an example configuration for that here. Bucket name, key, region can be updated of course. Note that this bucket will have to be manually created.

Reply:
> It requires us to persist the .tfstate file for the long-lived resources somewhere. (Though arguably we have to do so for the Lambda anyways.)

⬆️